### ISSUE TITLE
Get-HPOVServerProfile -Unassigned - v5.00 CMDLet

### ISSUE THREAD

**Issue by MSHAFFER64** [HewlettPackard/POSH-HPOneView#491]
The Get-HPOVServerProfile -Unassigned command is not producing a output of only the unassigned profiles.
It is lists all Profiles and Cannot validate argument on parameter 'Property'

Profile - BAY11-FC is a unassigned Profile.

Example:

```powershell
PS C:\Users\Administrator> Get-HPOVServerProfile -Unassigned
Where-Object : Cannot validate argument on parameter 'Property'. The argument is null or empty. Provide an argument that is not null or empty, and then try the command 
again.
At C:\Program Files\WindowsPowerShell\Modules\HPOneView.500\5.0.2400.1845\HPOneView.500.psm1:96655 char:82
+ ... esFromIndexCol = $_ResourcesFromIndexCol | Where-Object $null -eq ser ...
+                                                             ~~~~~
    + CategoryInfo          : InvalidData: (:) [Where-Object], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationError,Microsoft.PowerShell.Commands.WhereObjectCommand
 

Name      Status  Compliance Template    Server Hardware       Server Hardware Type   Enclosure Group         Affinity
----           ------ ----------      --------        ---------------          --------------------       --------------                 --------
BAY11-FC  OK     Unknown                                                        SY 480 Gen10 1              EG_3FRM                      Bay     
Bay5_FCoe OK     Unknown                   0000A66101, bay 5      SY 480 Gen9 2                EG_3FRM                      Bay     
```

**Comment by ChrisLynchHPE** [comment]
Apologies for the late reply to this issue.  Email notifications from GitHub appear to not be working.

Can you please provide the output from `$PSLibraryVersion | fl | clip`?

**Comment by ChrisLynchHPE** [comment]
This was addressed in the 5.20 and newer libraries.  If this is still a bug that needs to be addressed, please reply and this issue could be re-opened.

Closing due to no further activity.  If you wish, feel free to re-open to discuss further.
